src/placeos-pulse/message.cr

```diff
@@ -11,12 +11,19 @@ module PlaceOS::Pulse
     getter saas : Bool
     getter instance_id : String
     getter message : Request
+    getter signature : String
 
     def initialize(
       @instance_id : String,
       @saas : Bool,
-      @message : Request
+      @message : Request,
+      private_key : Sodium::Sign::SecretKey
     )
+      @signature = (private_key.sign_detached @message.to_json).hexstring
```

Comment:
```suggestion
      @signature = private_key.sign_detached(@message.to_json).hexstring
```

Reply:
Mainly suggesting this because it's more in line with the style used across the other codebases.
My reasoning is that more liberal use of parens makes for simpler chaining, and denotes calls in a slightly clearer fashion.